scripts/create-package/plop-templates-react/config/tests.js

```diff
@@ -1,7 +1 @@
 /** Jest test setup file. */
-
-const { configure } = require('enzyme');
```

Comment:
**q:** this will be used also for `node` packages right ?

Reply:
No they have a separate template, plop-templates-node

Reply:
I don't think node packages need enzyme configuration for tests? And even for v8 we're not using enzyme for new tests now.